src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -129,14 +129,30 @@ void SettingsLoader::GenerateProfiles()
 {
     const auto executeGenerator = [&](const auto& generator) {

```

Comment:
Here's a fun question!

Can you use SizeBench on a release build of TerminalSettingsModel and tell me whether the Template Foldability checker complains about this lambda? We're emitting five copies of it because it is differentiated based on type, and if those five copies aren't identical we might be wasting binary space :wink:

Reply:
Already did that and curiously it doesn't show up. IIRC the optimizer folds all invocations into a single function.
There's no binary size advantage in using the interface type either.